gensim/models/ldaseqmodel.py

```diff
@@ -130,7 +130,8 @@ def __init__(self, corpus=None, time_slice=None, id2word=None, alphas=0.01, num_
             if initialize == 'gensim':
                 lda_model = ldamodel.LdaModel(
                     corpus, id2word=self.id2word, num_topics=self.num_topics,
-                    passes=passes, alpha=self.alphas, random_state=random_state
+                    passes=passes, alpha=self.alphas, random_state=random_state,
+                    dtype=np.float64
```

Comment:
Maybe it will be a good idea to change default behaviour (to float32)?
CC @piskvorky @xelez

Reply:
Not now, `LdaSeqModel` will require modifications similar to those I made in `LdaModel` to handle dtype properly.